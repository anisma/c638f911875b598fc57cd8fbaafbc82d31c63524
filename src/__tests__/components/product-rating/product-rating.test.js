import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ProductRating from '../../../components/product-rating/product-rating.component';

describe('ProductRating Component', () => {
  

   const wrapper = shallow(
         <ProductRating />
   );
   it('should render with given state from Redux store', () => {
      expect(toJson(wrapper)).toMatchSnapshot();
   });
});
