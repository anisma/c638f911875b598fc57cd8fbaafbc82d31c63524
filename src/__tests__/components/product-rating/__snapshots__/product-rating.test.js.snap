// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductRating Component should render with given state from Redux store 1`] = `
<styled.div>
  <styled.span>
    NaN
  </styled.span>
  <StarBorderRoundedIcon
    fontSize="small"
    key="5"
    style={
      Object {
        "color": "#f9423a",
      }
    }
  />
  <StarBorderRoundedIcon
    fontSize="small"
    key="4"
    style={
      Object {
        "color": "#f9423a",
      }
    }
  />
  <StarBorderRoundedIcon
    fontSize="small"
    key="3"
    style={
      Object {
        "color": "#f9423a",
      }
    }
  />
  <StarBorderRoundedIcon
    fontSize="small"
    key="2"
    style={
      Object {
        "color": "#f9423a",
      }
    }
  />
  <StarBorderRoundedIcon
    fontSize="small"
    key="1"
    style={
      Object {
        "color": "#f9423a",
      }
    }
  />
</styled.div>
`;
